### DISCUSSION TITLE
Ajax populated select fields based on the values of other fields

### DISCUSSION THREAD

**Discussion by davinyoeun** [laravel/nova-issues#5907] [Q&A] [answered]
<img width="1451" alt="image" src="https://github.com/laravel/nova-issues/assets/129028087/f09f846b-01df-4f80-8eea-df3105166f7a">

**Comment by davidhemphill** [comment, marked as answer]
You're looking for [Dependent Fields](https://nova.laravel.com/docs/resources/fields.html#dependent-fields)
